packages/content/src/requests/items/formField/settings/RatingFormFieldSettings.ts

```diff
@@ -0,0 +1,33 @@
+import { serialize, type, validate } from "@js-soft/ts-serval";
+import { FormFieldSettings, FormFieldSettingsJSON, IFormFieldSettings } from "./FormFieldSettings";
+
+type MaxRating = 5 | 6 | 7 | 8 | 9 | 10;
+
+export interface RatingFormFieldSettingsJSON extends FormFieldSettingsJSON {
+    "@type": "RatingFormFieldSettings";
+    maxRating: MaxRating;
+}
+
+export interface IRatingFormFieldSettings extends IFormFieldSettings {
+    maxRating: MaxRating;
+}
+
+@type("RatingFormFieldSettings")
+export class RatingFormFieldSettings extends FormFieldSettings implements IRatingFormFieldSettings {
+    @serialize()
+    @validate({ min: 5, max: 10 })
+    public maxRating: MaxRating;
+
+    private readonly minRating = 1;
+    public getMinRating(): number {
+        return this.minRating;
+    }
```

Comment:
As in my comment this has to be static and also a getter, not a function named get... :)